packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1910,7 +1911,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         this.verifyNotClosed();
 
         if (this.context.pendingLocalState !== undefined) {
-            this.closeFn(CreateContainerError("op submitted while processing pending initial state"));
+            this.closeFn(new GenericError("op submitted while processing pending initial state"));
```

Comment:
wish this had some comments

Reply:
```suggestion
            this.closeFn(new GenericError("cotnainerRuntimeSubmitWithPendingLocalState"));
```

Reply:
> wish this had some comments

GitLens says @wes-carlson added this case, I think.  Wes feel free to drop a Suggestion comment in this PR and I'll include it